src/strava_py/cli.py

```diff
@@ -9,10 +9,17 @@ def main():
     parser.add_argument(
         "-o", "--output_file", default="plot.png", help="Output PNG file"
     )
+    parser.add_argument("lon_min", default=None, help="Minimum longitude for plot_map (values less than this are removed from the data)")
+    parser.add_argument("lon_max", default=None, help="Maximum longitude for plot_map (values greater than this are removed from the data)")
+    parser.add_argument("lat_min", default=None, help="Minimum latitude for plot_map (values less than this are removed from the data)")
+    parser.add_argument("lat_max", default=None, help="Maximum latitude for plot_map (values greater than this are removed from the data)")
+    parser.add_argument("alpha", default=0.4, help="Line transparency. 0 = Fully transparent, 1 = No transparency")
+    parser.add_argument("linewidth", default=0.4, help="Line width")
```

Comment:
Include `--` prefixes otherwise it complains about:

```
usage: strava_py [-h] [-o OUTPUT_FILE] path lon_min lon_max lat_min lat_max alpha linewidth
strava_py: error: the following arguments are required: lon_min, lon_max, lat_min, lat_max, alpha, linewidth
```


```suggestion
    parser.add_argument("--lon_min", default=None, help="Minimum longitude for plot_map (values less than this are removed from the data)")
    parser.add_argument("--lon_max", default=None, help="Maximum longitude for plot_map (values greater than this are removed from the data)")
    parser.add_argument("--lat_min", default=None, help="Minimum latitude for plot_map (values less than this are removed from the data)")
    parser.add_argument("--lat_max", default=None, help="Maximum latitude for plot_map (values greater than this are removed from the data)")
    parser.add_argument("--alpha", default=0.4, help="Line transparency. 0 = Fully transparent, 1 = No transparency")
    parser.add_argument("--linewidth", default=0.4, help="Line width")
```